ccvm_simulators/solvers/dl_solver.py

```diff
@@ -299,6 +302,8 @@ def _solve(
         # Start the timer for the solve
         solve_time_start = time.time()
 
+        c_sample = None # Make it global
+        s_sample = None # Make it global
```

Comment:
Can we not make them global now?

Reply:
Comments causes the confusion. So I removed all of them.